readme.md

```diff
@@ -76,9 +76,11 @@ cp -a myMusic myMedia
 **STOP DRAG AND DROP A FILE, OR CTRL + X, CTRL + V A FILE**
 
 ```
-mv readme.txt documents
+mv readme.txt documents/
 ```
 
+[For this reason](http://unix.stackexchange.com/a/50533)
```

Comment:
Yeah you're right, I did the PR fast and didn't notice that, thanks.
I think instead of that, I'd rather put the warning there because it's a common error that can cause a huge mess, what do you thinks of this?:

**Always** use a trailing slash when moving files, [for this reason](http://unix.stackexchange.com/a/50533)